components/producers/ossf-scorecard/Dockerfile

```diff
@@ -1,8 +1,3 @@
-ARG OSSF_SCORECARD_SAFETY_BASE_IMAGE
 FROM gcr.io/openssf/scorecard:stable
 
-FROM ${OSSF_SCORECARD_SAFETY_BASE_IMAGE}
-
-COPY --from=0 /scorecard /scorecard
-
-ENTRYPOINT ["/scorecard"]
\ No newline at end of file
+ENTRYPOINT ["/scorecard"]
```

Comment:
why is this needed any more? isn't the original image just invoking the `scorecard` binary?